bears/python/PyDocStyleBear.py

```diff
@@ -1,15 +1,36 @@
 from coalib.bearlib.abstractions.Lint import Lint
 from coalib.bears.LocalBear import LocalBear
+from coalib.settings.Setting import typed_list
 
 
 class PyDocStyleBear(LocalBear, Lint):
     executable = 'pydocstyle'
-    arguments = '{filename}'
     output_regex = r'(.*\.py):(?P<line>\d+) (.+):\n\s+(?P<message>.*)'
     use_stderr = True
 
-    def run(self, filename, file):
+    def run(self,
+            filename,
+            file,
+            pydocstyle_cli_options: str="",
+            pydocstyle_select: typed_list(str)=[],
+            pydocstyle_ignore: typed_list(str)=[]):
         '''
         Checks python docstrings.
+
+        :param pydocstyle_select:      List of checked errors by specifying
+                                       which errors to check for.
+        :param pydocstyle_ignore:      List of checked errors by specifying
+                                       which errors to ignore.
```

Comment:
@Makman2 I guess it's better if the user knows they are mutually exclusive when trying to use them together.